### PR TITLE
Disables random paperwork error.

### PR DESCRIPTION
**Pull request by Gladi-Writes**
## About The Pull Request

Disables paperwork error random event.

## How This Contributes To The Skyrat Roleplay Experience

Paperwork Error was too intrusive as a random event, a simple switch to admin-only will fix this. After a discussion with staff a few weeks ago consensus was reached that paperwork error is too intrusive to be a random event, especially given it can't be reversed for head-level jobs without admin intervention. Staff can still engage it when appropriate.

## Proof of Testing

Code identical to other blocked events, should work. 

## Changelog

:cl:
del: Paperwork Error random event.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 1]
```diff
@@ -125,3 +125,12 @@
...
 	max_occurrences = 0
+	
+/**
+ * Paperwork Error
+ *
+ * Removed:
+ *Paperwork Error is too intrusive and should be staff-only.
+ */
+/datum/round_event_control/bureaucratic_error
+    max_occurrences = 0
```
Comment (softcerv) [inline modular_skyrat/modules/events/code/event_overrides.dm:136]:
```suggestion
    max_occurrences = 0
    
```

File: modular_skyrat/modules/events/code/event_overrides.dm [hunk 2]
```diff
@@ -125,3 +125,12 @@
  */
 /datum/round_event_control/alien_infestation
 	max_occurrences = 0
+	
+/**
+ * Paperwork Error
+ *
+ * Removed:
+ *Paperwork Error is too intrusive and should be staff-only.
```
Comment (Zonespace27) [inline modular_skyrat/modules/events/code/event_overrides.dm]:
```suggestion
 * Paperwork Error is too intrusive and should be staff-only.
```

**Comment by Iamgoofball** [comment]
have you tried just using the HoP console

**Comment by Iamgoofball** [comment]
like, for the record, this is quite possibly one of the least intrusive random events, second to wise cow

if this is too intrusive, you may not want random events period

**Comment by Gladi-Writes** [comment]
> have you tried just using the HoP console

HOP console cannot reset head level jobs.

**Comment by Iamgoofball** [comment]
sounds like a bug, make an upstream report and we can fix it there

File: modular_skyrat/modules/events/code/event_overrides.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/events/code/event_overrides.dm:136]: Not really needed, now there's two empty lines at the end of the file.

**Comment by GoldenAlpharex** [comment]
> sounds like a bug, make an upstream report and we can fix it there

It's intentional at the HoP can't touch the amount of slots in the Heads job.

**Comment by Iamgoofball** [comment]
yes, but in the context of bureaucratic error, they should be able to

you can't raise head slots normally, but you should be allowed to lower them

**Review by tf-4 — APPROVED** [review]
this thing is annoying

**Pull request merged**